containers/Dockerfile.buildenv

```diff
@@ -210,6 +210,12 @@ RUN git clone --single-branch --branch ${CHARM_GIT_TAG} --depth 1 \
     && git apply /work/charm/v6.8.patch \
     && rm /work/charm/v6.8.patch \
     && rm -r /work/charm/doc /work/charm/examples
+RUN git clone https://github.com/UIUC-PPL/charm charm_6_10_2 \
+    && cd /work/charm_6_10_2 \
+    && git checkout v6.10.2 \
+    && ./build charm++ multicore-linux-x86_64 gcc ${PARALLEL_MAKE_ARG} -g -O0 \
+    && ./build charm++ multicore-linux-x86_64 clang ${PARALLEL_MAKE_ARG} -g -O0 \
+    && rm -r /work/charm/doc /work/charm/examples
```

Comment:
Here's the latest for this with the LIBS being built:
```
RUN git clone https://github.com/UIUC-PPL/charm charm_6_10_2 \
    && cd /work/charm_6_10_2 \
    && git checkout v6.10.2 \
    && ./build charm++ multicore-linux-x86_64 gcc ${PARALLEL_MAKE_ARG} -g -O0 \
    && ./build LIBS multicore-linux-x86_64 gcc ${PARALLEL_MAKE_ARG} -g -O0 \
    && ./build charm++ multicore-linux-x86_64 clang ${PARALLEL_MAKE_ARG} -g -O0 \
    && ./build LIBS multicore-linux-x86_64 clang ${PARALLEL_MAKE_ARG} -g -O0 \
    && rm -r /work/charm_6_10_2/doc /work/charm_6_10_2/examples
```
Please note that the `rm` portion is also update (the last line)